modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -3900,19 +4068,31 @@ Raft learner recovery rate limit. Throttles the rate of data communicated to nod
 
 ---
 
-=== raft_max_concurrent_append_requests_per_follower
+=== raft_max_buffered_follower_append_entries_bytes_per_shard
 
-Maximum number of concurrent append entry requests sent by the leader to one follower.
+The total size of append entry requests that may be cached per shard, using the Raft-buffered protocol. When an entry is cached the leader can continue serving requests because the ordering of the cached requests cannot change. When the total size of cached requests reaches the set limit, back pressure is applied to throttle producers.
```

Comment:
```suggestion
The total size of append entry requests that may be cached per shard, using the Raft-buffered protocol. When an entry is cached, the leader can continue serving requests because the ordering of the cached requests cannot change. When the total size of cached requests reaches the set limit, back pressure is applied to throttle producers.
```